src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NJsonSchema;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Test-Json command.
+    /// </summary>
+    [Cmdlet(VerbsDiagnostic.Test, "Json", HelpUri = "")]
+    public class TestJsonCommand : PSCmdlet
+    {
+        /// <summary>
+        /// An Json to be validated.
```

Comment:
Should be `A JSON string to be validated.`

Reply:
Fixed.